src/ast/value.rs

```diff
@@ -97,6 +97,37 @@ pub enum Value {
     Placeholder(String),
 }
 
+impl Value {
+    /// Get the string value of a `Value` wrapping a string.
+    /// This includes all quotation styles
+    /// `{Single,Double,TripleSingle,TripleDouble}{QuotedString,RawString,ByteString}`
+    /// as well as `EscapedStringLiteral`, `UnicodeStringLiteral`, `NationalStringLiteral`,
+    /// `HexStringLiteral`, `DollarQuotedString`.
```

Comment:
hmm seems it might not be worth enumerating the types after all since we have a lot of them. We could say something like
```text
/// If the underlying literal is a string, regardless of quote style, returns the associated string value
```